packages/terra-toolkit-docs/package.json

```diff
@@ -39,16 +39,16 @@
     "postinstall": "npm run compile"
   },
   "dependencies": {
-    "@cerner/jest-config-terra": "^1.6.1",
-    "@cerner/package-json-lint": "2.0.0",
-    "@cerner/stylelint-config-terra": "^4.4.1",
-    "@cerner/terra-functional-testing": "^4.2.1",
-    "@cerner/webpack-config-terra": "^3.1.1"
+    "@cerner/jest-config-terra": "^1.7.0",
+    "@cerner/package-json-lint": "^2.1.3",
```

Comment:
@sdadn Just have a minor question about if this is the version we want to specify since this won't be released until this is merged in an the CI workflow starts for this PR.

Reply:
Yeah it should be no different from the typical minor version bumps we do.